src/components/Table/rowRenderer.js

```diff
@@ -71,14 +75,30 @@ export default function defaultRowRenderer(
         <ExpandedComponent expandedData={data[openIndex]} />
       </div>
     );
-  } else if (index > openIndex) style.top = style.top + expandedHeight;
+  } else if (index > openIndex) {
+    style.top = style.top + expandedHeight;
+  }
+
+  // glyph hasn't been set, this isn't a selected row
+  if (glyph === undefined) glyph = 'fa-chevron-down';
```

Comment:
So this is more of a nitpick than anything else, but I feel like we could probably have this passed down through the theme. some sort of `expand` and `collapse` properties on the themes that would let people set custom icons (and the defaults would be chevron up and down)

Reply:
Ah ok, yeah I see where that would be a problem. It's one of the annoying things about aphrodite-css 😒. This is fine then. I suppose we could let someone pass in something like `glyph: { open, close }` and those could default to chevron up and down, but that's really just over engineering something that mainly just an aesthetic thing to have fun with more than anything useful